lib/node_modules/@stdlib/math/base/special/uimuldw/README.md

```diff
@@ -0,0 +1,89 @@
+<!--
+
+@license Apache-2.0
+
+Copyright (c) 2018 The Stdlib Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+-->
+
+# uimuldw
+
+> Compute the double word product of two unsigned 32-bit integers.
+
+<section class="intro">
+
+</section>
+
+<!-- /.intro -->
+
+<section class="usage">
+
+## Usage
+
+```javascript
+var uimuldw = require( '@stdlib/math/base/special/uimuldw' );
+```
+
+<!-- eslint-disable no-shortcut-reference-link -->
+
+#### uimuldw( \[out,\] a, b )
+
+> Multiplies two unsigned 32-bit integers and returns an array of two unsigned 32-bit integers which represents the 64-bit unsigned integer product.
+
+```javascript
+var v = uimuldw( 1, 10 );
+// returns [ 0, 10 ]
+
+v = uimul( 0x80000000, 0x80000000 ); // 2^31 * 2^31 = 4611686018427388000 => 32-bit integer overflow
+// returns [ 1073741824, 0 ]
+```
+
+</section>
+
+<!-- /.usage -->
+
+<!-- Package usage notes. Make sure to keep an empty line after the `section` element and another before the `/section` close. -->
```

Comment:
```suggestion
```

This is a stray comment which has wandered from its original place. You can just go ahead and remove.